src/backend/rest.ts

```diff
@@ -48,6 +49,13 @@ type RestApiGetRequest =
     }
   | CommonRequest;
 
+type RestApiFilterRequest =
+  | {
+      aggregation?: Aggregation;
+      filter: unknown;
+    }
+  | CommonRequest;
```

Comment:
This should be an "&" instead of a "|", shouldn't it?

But if it results in a large amount of errors that lead nowhere, you can leave it like that.

Reply:
i copy&pasted from the lines above which does the same for the get metrics request.

seems to work, i changed both of them.